src/js/modules/supervisors/Repository.ts

```diff
@@ -49,9 +49,9 @@ class Repository {
    * coprocessor. Returns undefined otherwise.
    * @param handle
    */
-  findByGlobalId = (handle: Handle): Handle | undefined =>
-    this.handles.get(handle.coprocessor.globalId);
-
+  findByGlobalId(handle: Handle): Handle | undefined {
```

Comment:
Can you explain why this needs to be done in the commit message body?

Reply:
because the arrow function doesn't add to the `Prototype` object, for that reason, I can mock it on my tests. also, the class function is better for performance. 

more information: https://medium.com/@charpeni/arrow-functions-in-class-properties-might-not-be-as-great-as-we-think-3b3551c440b1